scheduler/services/chronicle/data/WLGS.txt

```diff
@@ -0,0 +1,273 @@
+2018-02-1	21:13	00:38
+2018-02-1	04:18	06:13
+2018-02-2	23:35	00:01
+2018-02-2	05:34	06:26
+2018-02-5	21:35	06:20
+2018-02-6	21:34	06:21
+2018-02-8	21:32	00:32
```

Comment:
The dates here are a little hokey when the day number is a single digit (e.g. `2018-02-1`)... we should have these padded out to:
```
2018-02-01	04:18	06:13
2018-02-02	23:35	00:01
2018-02-02	05:34	06:26
2018-02-05	21:35	06:20
2018-02-06	21:34	06:21
2018-02-08	21:32	00:32
```
for example.